src/response/cache_sorted_set_fetch_response.rs

```diff
@@ -1,7 +1,7 @@
-use momento_protos::cache_client::SortedSetElement;
+use momento_protos::cache_client::sorted_set_fetch_response::SortedSet;
 
 #[derive(Debug)]
 #[non_exhaustive]
 pub struct MomentoSortedSetFetchResponse {
```

Comment:
I'm unsure if we really want this wrapper type. It doesn't appear to provide much value?

Reply:
@brayniac - FYI Kenny will be out on vacation for another week or so.

This PR may be relevant to your question:

https://github.com/momentohq/client-sdk-rust/pull/142

Kenny and I had started to have a conversation about how we will want to tweak the Rust SDK API on the way to 1.0, to make it more consistent with our other SDKs. He filed that PR to take a few of the APIs in the direction that made sense to him after our convo, and I think that our goal is for new APIs to look more like those (as well as going back and migrating the rest of the existing APIs to that pattern when we can, prior to 1.0).

I don't know if that answers your question? I have been meaning to spend some time in this repo to try to get ramped up on the delta between the "old" and "new" patterns that Kenny was trying to lay out, but I haven't had time yet so I don't know if I can give a great review.

That said, I'll try to budget some time for it later this week and see if I can grok enough to weigh in so that you don't have to wait all the way until Kenny returns.

Reply:
Thanks! That's helpful.